tests/test_port_dpb_system.py

```diff
@@ -225,3 +223,197 @@ def test_port_breakout_with_acl(self, dvs):
         self.dvs_acl.remove_acl_table("test")
         self.dvs_acl.verify_acl_table_count(0)
 
+    def test_dpb_arp_flush(self, dvs):
+        dvs.setup_db()
+        self.setup_db(dvs);
+
+        self.clear_srv_config(dvs)
+
+        # Create l3 interface
+        rif_oid = self.create_l3_intf("Ethernet0", "")
+
+        # set ip address
+        self.add_ip_address("Ethernet0", "10.0.0.0/31")
+
+        # bring up interface
+        self.set_admin_status(dvs, "Ethernet0", "up")
+
+        # Set IP address and default route
+        dvs.servers[0].runcmd("ip address add 10.0.0.1/31 dev eth0")
+        dvs.servers[0].runcmd("ip route add default via 10.0.0.0")
+
+        # Get neighbor and ARP entry
+        dvs.servers[0].runcmd("ping -c 1 10.0.0.0")
+
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_NEIGHBOR_ENTRY")
+        intf_entries = tbl.getKeys()
+        assert len(intf_entries) == 1
+        route = json.loads(intf_entries[0])
+        assert route["ip"] == "10.0.0.1"
+        assert route["rif"] == rif_oid
+        (status, fvs) = tbl.get(intf_entries[0])
+        assert status == True
+
+        fvs_dict = dict(fvs)
+        assert fvs_dict["SAI_NEIGHBOR_ENTRY_ATTR_DST_MAC_ADDRESS"] == "46:C1:84:2C:3B:4A"
+
+        # Breakout port and make sure NEIGHBOR entry is removed
+        dvs.verify_port_breakout_mode("Ethernet0", "1x100G[40G]")
+        dvs.change_port_breakout_mode("Ethernet0", "4x25G[10G]", "-f")
+        dvs.verify_port_breakout_mode("Ethernet0", "4x25G[10G]")
+
+        #Verify ARP/Neighbor entry is removed
+        intf_entries = tbl.getKeys()
+        assert len(intf_entries) == 0
+
+        dvs.change_port_breakout_mode("Ethernet0", "1x100G[40G]")
+        dvs.verify_port_breakout_mode("Ethernet0", "1x100G[40G]")
+
+    def test_dpb_vlan_arp_flush(self, dvs):
```

Comment:
Since we already have this, to cover the code changes, can we add two more test cases: 1, shutdown port.  2, remove port from vlan.  To verify the ARP was being flushed.

Reply:
Make sense. But I feel they should be part of test_port.py and test_vlan.py respectively!But yeah, I can add them here and move them later. What do you say?

Reply:
OK